### PR TITLE
Set to wrong inasafe_data path to see which tests are failing.

### PR DESCRIPTION
**Pull request by ismailsunni**
Don't merge it

### CODE REVIEW

**Comment by Gustry** [comment]
Can we update this PR with develop ? To see which tests are using inasafe_data.

**Comment by ismailsunni** [comment]
Updated.

**Comment by Gustry** [comment]
Thanks
Ouch, there are a lot of failing tests again !
Let's wait after 4.0 ... ;-)

**Comment by Gustry** [comment]
FAILED (failures=3, errors=29, skipped=77, expected failures=27)

**Comment by ismailsunni** [comment]
FAILED (failures=3, errors=17, skipped=39, expected failures=30, unexpected successes=1)

**Comment by Gustry** [comment]
inasafe_data has been removed from travis in develop already ! 👍

**Comment by timlinux** [comment]
Woohooo!

**Comment by ismailsunni** [comment]
So we can close this PR? @Gustry

File: safe/test/utilities.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
     '..'))
 
 # Location of test data
-DATANAME = 'inasafe_data'
```
Comment (Gustry) [inline safe/test/utilities.py]: We need to remove this kind of code in develop.

Comment (ismailsunni) [inline safe/test/utilities.py]: Ah yes, we still have some test that use inasafe_data, but they are marked as expectedFailure

**Pull request closed without merging**